tests/e2e/specs/components/web/ux/label/textWrapTrue.spec.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  tags: ['component', 'label'],
+  before: function(client, done) {
+    this.currentPage = client.maximizeWindow().page.labelPage();
+    this.currentPage
+      .navigate('http://localhost:8080/components/label.html')
```

Comment:
can we set domain in a shared variable? 'localhost:8080' might change in the future

Reply:
domain set in globals.js